### DISCUSSION TITLE
请问怎么样才能通过qq号获取用户昵称？或者哪个函数可以像get_user_id一样获取用户的昵称？

### DISCUSSION THREAD

**Discussion by Lkhsss** [nonebot/discussions#185] [问答] [answered]
### 请问我该怎么做？

**Comment by yanyongyu** [comment, marked as answer]
如果你是要群昵称的话在event.sender里应该有，如果你是要用户QQ名称的话得调用api `get_stranger_info`

**Reply by Lkhsss** [reply]
> 如果你是要群昵称的话在event.sender里应该有，如果你是要用户QQ名称的话得调用api `get_stranger_info`

# 谢谢！
### event.sender获取群昵称的方法我成功了，但是调取api的方法却不知道怎么做，请问可以给一个示例吗？

**Reply by yanyongyu** [reply]
看文档里调用平台api一节

**Reply by Lkhsss** [reply]
> 看文档里调用平台api一节

## 我已仔细观看文档， 使用了文档中的api调用方法，nb报错为`nonebot.adapters.onebot.exception.ActionFailed: <ActionFailed data=None, echo=1, msg=API_NOT_FOUND, retcode=404, status=failed, wording=API不存在>`

## 我的报错：
## 1.使用`await bot.call_api("get_user_info", user_id=xxxxxxxxx)`方法
  报错为`nonebot.adapters.onebot.exception.ActionFailed: <ActionFailed data=None, echo=1, msg=API_NOT_FOUND, retcode=404, status=failed, wording=API不存在>`
## 2.使用`result = await bot.get_user_info(user_id=xxxxxxxxx)`方法
 报错为`nonebot.adapters.onebot.exception.ActionFailed: <ActionFailed data=None, echo=1, msg=API_NOT_FOUND, retcode=404, status=failed, wording=API不存在>`
> 注：两种方法使用前均已实例化
### 救救孩子吧，孩子只是想用get_status返回一下服务状态

**Reply by yanyongyu** [reply]
你的api名称是错的，是`get_stranger_info`

**Comment by wei-z-git** [comment]
有啥能获得用户性别的方法吗，get_group_member_list返回的是unknown
